Sources/Redbird/Redbird.swift

```diff
@@ -184,3 +184,17 @@ struct CommandFormatter {
     }
 }
 
+// Add support for pub-sub
```

Comment:
It's not immediately clear what this adds to allow for pub sub, can you expand this comment a bit?